micro_dl/evaluation/evaluation.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+import micro_dl.inference.evaluation_metrics as inference_metrics
+from torch.utils.tensorboard import SummaryWriter
+
+
+class TorchEvaluator(object):
+    """
+    Handles all procedures involved with model evaluation.
+
+    Params:
+    :param dict torch_config: master config file
+    """
+    def __init__(self, torch_config, device=None) -> None:
+        self.torch_config = torch_config
+
+        self.zarr_dir = self.torch_config["zarr_dir"]
+        self.network_config = self.torch_config["model"]
+        self.training_config = self.torch_config["training"]
+        self.dataset_config = self.torch_config["dataset"]
+        self.inference_config = self.torch_config["inference"]
+        self.preprocessing_config = self.torch_config["preprocessing"]
+
+        self.inference_metrics = {}
+        self.log_writer = SummaryWriter(log_dir=self.save_folder)
+    
+    def get_save_location(self):
+        """
+        Sets save location as specified in config files.
+        """
+        #TODO implement
```

Comment:
Maybe throw an error instead?

```suggestion
        #TODO implement
```
```py
raise NotImplementedError
```